R/PKNCA_impute_method_additions.R

```diff
@@ -0,0 +1,29 @@
+PKNCA_impute_method_start_log <- function(conc, time, start, end, ..., options = list()) {
```

Comment:
Hello @m-kolomanski ! I was wondering, the variable names of these functions all should start with "PKNCA_impute_method" due to PKNCA package restrictions to use them in their methods. Is it ok if I just add the names of these functions to the WORDLIST file? Will that actually make the work to pass the lint and spelling tests?

`Line 1[object_name_linter] Variable and function name style should match snake_case or SNAKE_CASE.`

Reply:
Hey,

WORDLIST works only for spellchecking, does not impact the code on linter. If you wish to stop linter from evaluating particular line, use `# nolint` comment, so:
```R
PKNCA_impute_method_start_log <- function(conc, time, start, end, ..., options = list()) { # nolint
```
Would you mind providing me with the relevant docs for why that is the case?  I have failed to find anything related and I am unsure what functionality we want to use exactly.

Reply:
Hello, is for keeping up to date with the data imputation functions that the package provides and the way it handles them, you can create your own function using the same notation, check the last section of [this vignette](https://cran.r-project.org/web/packages/PKNCA/vignettes/v08-data-imputation.html)